forbidigo/forbidigo.go

```diff
@@ -58,7 +58,7 @@ type Linter struct {
 }
 
 func DefaultPatterns() []string {
-	return []string{`^((?P<pkg>fmt)\.Print(|f|ln)|print|println)$`}
```

Comment:
This didn't work for the case without a package name.

Reply:
Which Go code contains just plain `print` or `println` function calls? Do we really need to support that?

Reply:
`print` and `println` are built-in go functions (see https://go.dev/ref/spec) that provide an easy way to output log lines (but are generally bad news for production code)

Reply:
(I haven't mentioned it to this point but I have been wondering it would be useful to provide some way to differentiate between built-in functions and local redefinitions, but it didn't seem important enough to me to worry about for now)

Reply:
Interesting, I didn't know those functions exist - my introduction to Go skipped over them, obviously for good reasons :grin: 

I copied this over into the main PR.